e2e/cypress/integration/ui/settings/04_project_mgmt.spec.ts

```diff
@@ -0,0 +1,206 @@
+interface Project {
+  id: string;
+  name: string;
+  type: string;
+}
+
+// assume 2.0 if not in CI. if you wish something different start cypress with
+// IAM_VERSION set to what you are testing.
+if (Cypress.env('IAM_VERSION') === undefined) {
+  Cypress.env('IAM_VERSION', 'v2.0');
+}
+
+const describeIAMV2P1 = Cypress.env('IAM_VERSION') === 'v2.1' ? describe : describe.skip;
+
+describeIAMV2P1('project management', () => {
+  let adminToken = '';
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'cypress-test';
+  const projectID = `${cypressPrefix}-project1-${now}`;
+  const projectName = `${cypressPrefix} project1 ${now}`;
+  const ruleID = `${cypressPrefix}-rule-${now}`;
+  const ruleName = `${cypressPrefix} rule ${now}`;
+
+  before(() => {
+    cy.adminLogin('/settings/projects').then(() => {
+      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+      adminToken = admin.id_token;
+      cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
+    });
+  });
+
+  beforeEach(() => {
+    cy.restoreStorage();
+  });
+
+  afterEach(() => {
+    cy.saveStorage();
+  });
+
+
+  it('list projects', () => {
```

Comment:
For consistency, need more words, e.g.:
```suggestion
  it('displays a list of projects', () => {
```